source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -82,13 +134,32 @@ bool AggressiveDCEPass::AllExtensionsSupported() const {
   return true;
 }
 
+bool AggressiveDCEPass::IsDead(ir::Instruction* inst) {
+  if (IsLive(inst)) return false;
+  if (inst->IsBranch() &&
+      !IsStructuredIfOrLoopHeader(context()->get_instr_block(inst), nullptr,
```

Comment:
The method comment for IsStructuredIfOrLoopHeader should say that any of the last three args could be nullptr.
(Pre-existing problem.)

Reply:
Will change.